client/paper/entry.js

```diff
@@ -119,6 +136,9 @@ import { fillQuadTex, fillTriTex } from './canvasUtils';
     callback,
     whiskerPointCallback,
   } = {}) => {
+    // eslint-disable-next-line no-console
+    console.warn('paper.whenPointsAt is deprecated, use the new whisker api instead');
```

Comment:
I don't know how we should handle deprecation of features. I've kept the original whiskers for now, but long term I think we should remove `paper.whenPointsAt` to avoid confusion with the new whisker api.

Reply:
Can we remove the code below and call `get('whisker')` when calling this? So we don't have any duplicate code at least?

The `console.warn` makes a lot of sense, let's keep that.